docs/integrations/create-graphiql-plugin.md

```diff
@@ -0,0 +1,142 @@
+# Graphiql custom plugin
+
+You can easily create a graphiql plugin and integrate it in mercurius graphiql instance.
+
+[More info here.](https://github.com/graphql/graphiql)
+
+[GraphiQL Explorer Plugin example.](https://github.com/graphql/graphiql/tree/main/packages/graphiql-plugin-explorer)
+
+## Create the plugin
+
+### Plugin component
+
+A Graphiql plugin is an object that exports these values:
```

Comment:
please keep the naming of graphiql consistent throughout. Use the official name when possible, either upper case or lower, but avoid mixing them, for consistency

Reply:
also note that the wording "exports" is inaccurate. it would be accurate if it referred to a module, but an object does not "export" values, it "contains properties"